hbase-commit-table/src/main/java/com/yahoo/omid/committable/hbase/HBaseCommitTable.java

```diff
@@ -86,9 +87,11 @@ protected HBaseCommitTable(Configuration hbaseConfig, String tableName, KeyGener
 
     public class HBaseWriter implements Writer {
 
+        private static final long INITIAL_LWM_VALUE = -1L;
         final HTable table;
         // Our own buffer for operations
         final List<Put> writeBuffer = new LinkedList<>();
+        final AtomicLong lowWatermarkToStore = new AtomicLong(INITIAL_LWM_VALUE);
```

Comment:
I'va also added a test that I had in my local repo that ensures that persistLowWatermark() is called only when an element is evicted from the cache. This tests the new functionality added in this PR.